source/state_representation/src/State.cpp

```diff
@@ -33,14 +33,10 @@ void State::set_filled() {
   this->reset_timestamp();
 }
 
-const std::chrono::time_point<std::chrono::steady_clock>& State::get_timestamp() const {
+const std::chrono::time_point<std::chrono::steady_clock>& State::get_epoch() const {
   return this->timestamp_;
```

Comment:
If we use seconds for convenience, you can cast it as follows: 
```c++
  std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::steady_clock::now() - this->timestamp_).count() * 1e9
```

Reply:
Dividing by 1e9 right?

Reply:
ah yes, and also `static_cast<double>`. I mainly just wanted to write the `duration_cast` down because it's easy to forget and I always have to look it up